examples/neural_compressor/question-answering/run_qa.py

```diff
@@ -513,6 +543,84 @@ def prepare_train_features(examples):
             # Number of samples might increase during Feature Creation, We select only specified max samples
             train_dataset = train_dataset.select(range(data_args.max_train_samples))
 
+    accelerator = Accelerator(cpu=training_args.no_cuda)
+
+    def move_input_to_device(input, device):
+        if isinstance(input, torch.Tensor):
+            return input if input.device == device else input.to(device)
+        elif isinstance(input, tuple):
+            return tuple([move_input_to_device(ele, device) for ele in input])
+        elif isinstance(input, list):
+            return [move_input_to_device(ele, device) for ele in input]
+        elif isinstance(input, dict):
+            return {key: move_input_to_device(input[key], device) for key in input}
+        else:
+            assert False, "only support input type of torch.Tensor, tuple, list and dict."
+
+    # get logits of teacher model
+    if optim_args.run_teacher_logits:
+        assert data_args.pad_to_max_length, "to run teacher logits must open pad_to_max_length due to padding issue"
+        teacher_config = AutoConfig.from_pretrained(optim_args.teacher_model_name_or_path)
+        teacher_model = AutoModelForQuestionAnswering.from_pretrained(
+            optim_args.teacher_model_name_or_path,
+            from_tf=bool(".ckpt" in optim_args.teacher_model_name_or_path),
+            config=teacher_config,
+        )
+        teacher_model = QAModel_output_reshaped(teacher_model)
+        teacher_model = accelerator.prepare(teacher_model)
+        para_counter = lambda model: sum(p.numel() for p in model.parameters())
```

Comment:
What do you think about 
```suggestion
        num_param = lambda model: sum(p.numel() for p in model.parameters())
```